apis/v1alpha1/generator.yaml

```diff
@@ -56,8 +56,8 @@ resources:
     hooks:
       delta_pre_compare:
         code: customPreCompare(delta, a, b)
-      sdk_update_post_request:
-        code: return desired, nil
```

Comment:
I think you should add to the description something about this and why it doesn't work also you can state that I do not recall why I did it..